course04/homework01/arne/main.cc

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <array>
+#include <algorithm>
+
+using namespace std;
```

Comment:
This isn't a good idea - in a .cc file I guess it's questionable, in a header file it should never be used. At Optiver we never do this, so please stick to this convention